src/main.cpp

```diff
@@ -745,6 +745,32 @@ int64 GetMinFee(const CTransaction& tx, bool fAllowFree, enum GetMinFee_mode mod
 }
 
 
+static
```

Comment:
How often do I need to tell you to NOT do this in 2 lines ;)? @laanwj Also did this...